.env.example

```diff
@@ -1,8 +1,8 @@
 AM_URL=https://default.iam.example.com/am
 BASE_URL=https://example.com
 CLIENT_ID=a330c848f3f2953eb543d954527a7d0e
-PASSWORD=correct horse battery staple
 REALM_PATH=root
 SCOPE=openid profile
 TREE=UsernamePassword
 USERNAME=xkcd
+PASSWORD=correct horse battery staple
```

Comment:
Haha, don't like alphabetization when it breaks up username and password?

Reply:
Ha ha ha, you're perceptive! Yeah, it will killing me to have password separated. I can change it ... it's just my mind playing with me.

Reply:
I don't entirely disagree.  But every time I run across something like this, I realize I'm on a slippery slope to inconsistency based on fairly arbitrary decisions.

Reply:
:) I'll change it back